examples/testify/src/integration-test/groovy/gorm/tools/repository/BulkableRepoIntegrationSpec.groovy

```diff
@@ -31,9 +34,15 @@ class BulkableRepoIntegrationSpec extends Specification implements DomainIntTest
     JdbcTemplate jdbcTemplate
     OrgRepo orgRepo
     KitchenSinkRepo kitchenSinkRepo
+    @Inject QueryConfig queryConfig
```

Comment:
is @Inject needed here for config to work?

Reply:
Cleaned up